DEVELOPERS.md

```diff
@@ -68,6 +68,18 @@
          `Console` tab to view printed logs and errors, the `Element` tab to
          inspect the HTML and CSS, the `Network` tab for network calls, etc).
 
+#### Debugging Add-in: Alternate way
+If the above method to debug the add-in is not working, one can try the following steps:
+   1. Delete the `C:\Users{username}.office-addin-dev-certs` folder.
+   2. Run `npx office-addin-dev-certs install`.
+   3. Now run `npm run dev-server` to startup the server and `npm run start` 
+      to open the Add-in in Word.
+   4. Click on Add-in to open it.
+   5. Instead of attaching the debugger from add-in, open VS Code and click on 
+      `Debug and run`. Click on the green arrow in top left corner that says
+      `Start Debugging`. A popup may appear. Select debug anyway. The debugger is 
```

Comment:
If you add the file as I commented in the previous comment then there would be no need to do this : `Select debug anyway.`